### DISCUSSION TITLE
Taiko --emulate-device 'iPhone X' does not show the view port

### DISCUSSION THREAD

**Discussion by harimadusumilli** [getgauge/taiko#1868] [General]
On a windows 10 machine. AT the command prompt.

I typed the following Taiko --emulate-device 'iPhone X' to open the view port. I get the following message displayed instead.

**taiko --emulate-device 'iPhone X'**
**Invalid value 'iPhone for --emulate-device**
Available devices: Blackberry PlayBook, Blackberry PlayBook landscape, BlackBerry Z30, BlackBerry Z30 landscape, Galaxy Note 3, Galaxy Note 3 landscape, Galaxy Note II, Galaxy Note II landscape, Galaxy S III, Galaxy S III landscape, Galaxy S5, Galaxy S5 landscape, iPad, iPad landscape, iPad Mini, iPad Mini landscape, iPad Pro, iPad Pro landscape, iPhone 4, iPhone 4 landscape, iPhone 5, iPhone 5 landscape, iPhone 6, iPhone 6 landscape, iPhone 6 Plus, iPhone 6 Plus landscape, iPhone 7, iPhone 7 landscape, iPhone 7 Plus, iPhone 7 Plus landscape, iPhone 8, iPhone 8 landscape, iPhone 8 Plus, iPhone 8 Plus landscape, iPhone SE, iPhone SE landscape, iPhone X, iPhone X landscape, Kindle Fire HDX, Kindle Fire HDX landscape, LG Optimus L70, LG Optimus L70 landscape, Microsoft Lumia 550, Microsoft Lumia 950, Microsoft Lumia 950 landscape, Nexus 10, Nexus 10 landscape, Nexus 4, Nexus 4 landscape, Nexus 5, Nexus 5 landscape, Nexus 5X, Nexus 5X landscape, Nexus 6, Nexus 6 landscape, Nexus 6P, Nexus 6P landscape, Nexus 7, Nexus 7 landscape, Nokia Lumia 520, Nokia Lumia 520 landscape, Nokia N9, Nokia N9 landscape, Pixel 2, Pixel 2 landscape, Pixel 2 XL, Pixel 2 XL landscape

I am not sure as to how else to do this or is this is the right way to do it. Any help is appreciated.

Thanking you in advance.

**Comment by saikrishna321** [comment]
@harimadusumilli Works for me. 

![Screenshot 2021-02-17 at 9 58 23 AM](https://user-images.githubusercontent.com/4045041/108156472-bb3cd280-7106-11eb-9efc-e50812bbd1cb.png)

**Comment by harimadusumilli** [comment]
I am working with on a Windows 10 machine. Is there anything that I need to configure or check. I see that your example or screen shot you have provided me is on a mac machine. Any help is appreciated. As, I am at a blocker right now and don't seem to have any idea.
>taiko -v
Version: 1.2.0 (Chromium: 88.0.4314.0) RELEASE

**Comment by NivedhaSenthil** [comment]
Can someone who has access to Windows machine help verifying this issue ?

**Reply by srp-at-ema** [reply]
I cannot reproduce it @harimadusumilli (Windows 10, taiko 1.20)
